### PR TITLE
Composer config github-oauth

### PR DESCRIPTION
**Pull request by thiagotalma**
Add the possibility of including the token used on Github.

According to the instructions here: https://getcomposer.org/doc/articles/troubleshooting.md#api-rate-limit-and-oauth-tokens

### CODE REVIEW

File: roles/composer/tasks/main.yml [hunk 1]
```diff
@@ -16,3 +16,7 @@
     regexp: ^PATH="(((?!:./vendor/bin).)*)"
     line: PATH="\1:./vendor/bin"
     backrefs: yes
+
+- name: Composer config github-oauth
+  command: composer config -g github-oauth.github.com {{ composer_githuboauth }}
+  when: composer_githuboauth is defined
```
Comment (swalkinshaw) [inline roles/composer/tasks/main.yml]: @thiagotalma little nitpick, but could the variable be called `composer_github_oauth_token`?

**Comment by retlehs** [comment]
some details on what this is for would be helpful for people coming to look at this pull request

**Comment by thiagotalma** [comment]
@swalkinshaw done!

@retlehs I'm sorry I have not done before.
I imagined that the title would be self-descriptive.

**Comment by swalkinshaw** [comment]
@thiagotalma would you be able to squash these two commits into 1?

Guide if you need it: https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

**Comment by swalkinshaw** [comment]
Landed in 625ca3f1417476a29dc8bef85ba2d92044c0883e

**Pull request closed without merging**